src/hub.ts

```diff
@@ -268,7 +287,7 @@ export class Hub extends EventEmitter {
         if (port.connected) {
             port.type = type;
             if (this.autoSubscribe) {
-                this._activatePortDevice(port.value, type, this._getModeForDeviceType(type), 0x00);
+                this.subscribe(port.id, this._getModeForDeviceType(type));
```

Comment:
This changed only to have the exact same result using manual subscribe or autosubscribe.